### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by timuric**

### CODE REVIEW

**Review by 2can — APPROVED** [review]

File: docs/developer/payments.mdx [hunk 1]
```diff
@@ -669,7 +669,7 @@ The webhook will be sent only to the app that is the owner of the transaction.
 
 :::
 
-##### Asynchronously processing the action
```
Comment (patrys) [inline docs/developer/payments.mdx:672]: This looks like an unrelated change.

File: docusaurus.config.js [hunk 2]
```diff
@@ -348,8 +348,8 @@ module.exports = {
     image: "img/og-image.png",
 
     prism: {
-      theme: require("prism-react-renderer/themes/github"),
-      darkTheme: require("prism-react-renderer/themes/dracula"),
+      theme: require("prism-react-renderer/themes/oceanicNext"),
+      defaultLanguage: "graphql",
```
Comment (patrys) [inline docusaurus.config.js:352]: Why set a default language? I'd rather explicitly mark all fenced code examples.

File: src/theme/SearchBar/index.js [hunk 3]
```diff
@@ -1,8 +1,23 @@
...
+  //Radix color variables require a class on the root element
+  useEffect(() => {
+    //Add classes for radix color variables
+    if (colorMode === "dark") {
+      document.documentElement.classList.add("dark-theme");
+    } else {
+      document.documentElement.classList.remove("dark-theme");
+    }
+  }, [colorMode]);
+};
```
Comment (patrys) [inline src/theme/SearchBar/index.js:18]: It's a hack. Please add documentation on why it's here and how it works. I'm afraid that it will stop working one day and no one will be able to understand how theming ever worked.

**Review by patrys — CHANGES_REQUESTED** [review]

**Review by zaiste — APPROVED** [review]

File: docs/developer/payments.mdx (hunk 1 above)
Comment (timuric) [reply docs/developer/payments.mdx:672]: It is, as h5 fives should not be used as we don't differentiate h5 and h4, and those are actually typos and should be h4

File: docusaurus.config.js (hunk 2 above)
Comment (timuric) [reply docusaurus.config.js:352]: Unintended change, thanks!

File: src/theme/SearchBar/index.js (hunk 3 above)
Comment (timuric) [reply src/theme/SearchBar/index.js]: This hack was removed in later commit

**Review by patrys — APPROVED** [review]
